app/controllers/returns/credits/ExportedCreditsController.scala

```diff
@@ -45,7 +45,12 @@ class ExportedCreditsController @Inject()
 
   def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) {
     implicit request =>
-      Ok(view(formProvider(), mode))
+      val userAnswers = request.userAnswers
+      val preparedForm = userAnswers.get(ExportedCreditsPage) match {
+        case None => formProvider()
+        case Some(value) => formProvider().fill(value)
+      }
```

Comment:
```suggestion
      val preparedForm = request.userAnswers.fill(ExportedCreditsPage, formProvider())
```

Reply:
it's on another commit (along with some unfinished tests) I haven't pushed yet :(